packages/web-components/src/option/README.md

```diff
@@ -0,0 +1,139 @@
+| `value`             | public    | `string`                                                                                                       |         |                                                                                                                                                                                      |
+| `form`              | public    | `HTMLFormElement or null`                                                                                      |         |                                                                                                                                                                                      |
+| `proxy`             |           | <code>new Option(&#96;${this.textContent}&#96;, this.initialValue, this.defaultSelected, this.selected)</code> |         |                                                                                                                                                                                      |
+
+<br />
+
+### **Methods**
+
+<br />
+
+| Name                       | Privacy   | Description                                                         | Parameters                                | Return | Inherited From |
+| -------------------------- | --------- | ------------------------------------------------------------------- | ----------------------------------------- | ------ | -------------- |
+| `checkedChanged`           | public    | Updates the ariaChecked property when the checked property changes. | `prev: boolean or unknown, next: boolean` | `void` |                |
+| `defaultSelectedChanged`   | protected |                                                                     |                                           | `void` |                |
+| `disabledChanged`          | protected |                                                                     | `prev: boolean, next: boolean`            | `void` |                |
+| `selectedAttributeChanged` | protected |                                                                     |                                           | `void` |                |
+| `selectedChanged`          | protected |                                                                     |                                           | `void` |                |
+| `initialValueChanged`      | public    |                                                                     | `previous: string, next: string`          | `void` |                |
+
+<br />
+
+### **Attributes**
+
+<br />
+
+| Name       | Field             |
+| ---------- | ----------------- |
+|            | disabled          |
+| `selected` | selectedAttribute |
+| `value`    | initialValue      |
```

Comment:
I see attributes here now...I think this needs to get closer to the "fields" portion as these are fundamentally linked. Can we just merge those two? It's confusing seeing them separate and then repeated.